tests/vmi_configuration_test.go

```diff
@@ -1165,7 +1164,7 @@ var _ = Describe("Configurations", func() {
 				})
 
 				It("[test_id:]VMI condition should signal unsupported agent presence", func() {
-					tests.SkipIpv6TestsFailingAfterVmImageUpgradeToFc32()
+
```

Comment:
there are some blank lines that were added in this file, if you want to remove please

Reply:
Done